### PR TITLE
SRX Branch cluster fails SW.install().

### PR DESCRIPTION
**Pull request by stacywsmith**
On SRX branch devices configured in an SRX cluster, `SW.install()`
fails because it recognizes the cluster as `'2RE' == True`, and attempts
to upgrade both REs. However, this product doesn't not support the
`re0` and `re1` arguments.

This change avoids attempting to upgrade both REs of the cluster,
even when `all_re = True`.

### CODE REVIEW

**Comment by stacywsmith** [comment]
Addresses Juniper/ansible-junos-stdlib#261

File: lib/jnpr/junos/utils/sw.py [hunk 1]
```diff
@@ -967,7 +972,7 @@ def reboot(self, in_min=0, at=None, all_re=True):
         else:
             cmd = E('request-reboot', E('at', str(at)))
 
-        if all_re is True:
+        if all_re is True :
```
Comment (vnitinv) [inline lib/jnpr/junos/utils/sw.py]: why extra space?

File: lib/jnpr/junos/utils/sw.py [hunk 2]
```diff
@@ -72,6 +72,11 @@ def __init__(self, dev):
             self._RE_list = [x for x in dev.facts.keys()
                              if x.startswith('version_RE')]
         self._multi_RE = bool(dev.facts.get('2RE'))
+        # Branch SRX in an SRX cluster doesn't really support multi_RE
+        # functionality for SW.
+        if (dev.facts.get('personality', '') == 'SRX_BRANCH' and
```
Comment (vnitinv) [inline lib/jnpr/junos/utils/sw.py]: This change won't send re0 to rpc call, but do we need to install for different node explicitly.

File: lib/jnpr/junos/utils/sw.py (hunk 1 above)
Comment (stacywsmith) [reply lib/jnpr/junos/utils/sw.py]: That was an accident. Thanks for catching it. I have corrected it.

File: lib/jnpr/junos/utils/sw.py (hunk 2 above)
Comment (stacywsmith) [reply lib/jnpr/junos/utils/sw.py]: Unfortunately, this platform/configuration does not support the ability to upgrade a node other than the currently connected node. Therefore, it must be the user's responsibility to explicitly perform the `SW.install()` on each node of the cluster.

**Pull request merged**